src/devices/src/virtio/balloon/test_utils.rs

```diff
@@ -1,13 +1,17 @@
 // Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+#![doc(hidden)]
+
 use std::u32;
 
 use crate::virtio::test_utils::VirtQueue;
+#[cfg(test)]
 use crate::virtio::{
     balloon::NUM_QUEUES, Balloon, IrqType, DEFLATE_INDEX, INFLATE_INDEX, STATS_INDEX,
 };
 
+#[cfg(test)]
```

Comment:
Nit: for consistency and readability reasons, we should have all test functions at the end of the file.